leku/src/main/java/com/schibstedspain/leku/geocoder/GeocoderPresenter.java

```diff
@@ -56,6 +59,7 @@ public void getFromLocationName(String query) {
         .subscribeOn(Schedulers.newThread())
         .observeOn(scheduler)
         .retry(RETRY_COUNT)
+        .onErrorResumeNext(apiInteractor.getFromLocationName(query))
```

Comment:
At the end is a library, not a full app, so it's ok to have "less" layers (IMHO). This implementation makes swapping them easy peasy